packages/nx-sonarqube/src/executors/scan/utils/utils.ts

```diff
@@ -35,12 +35,17 @@ class ExtraMarshaller implements OptionMarshaller {
   }
 }
 class EnvMarshaller implements OptionMarshaller {
+  private readonly normalizedOptionsName = {
+    ['sonar.projectversion']: 'sonar.projectVersion',
```

Comment:
Project version is already in the [schema](https://github.com/koliveira15/nx-sonarqube/blob/640fc4666c5dd19f02ad1373b5b8698ff831ba30/packages/nx-sonarqube/src/executors/scan/schema.d.ts#L8).

Reply:
Yeah. But I only know the project version when running the build. So, I can't use the env variable there.

What do you recommend?

Reply:
@hermogenes If you need to pass a dynamic projectVersion, I would do so via the Nx executor command and pass the version from your CI build to it like so:

```
npx nx sonar my-app --projectVersion 1.2.3 
```
or
```
npx nx affected --target sonar --projectVersion 1.2.3
```

Reply:
Obrigado @koliveira15!

Use NX command works 

I just tried to make the lib work according to docs (and it says that properties are configurable via SONAR_ vars which is not true for some of them).

Anyways, thanks and will close this PR